### ISSUE TITLE
Persistent Unchecking of 'Disable Dropdowns' in Gantry 5 Menu Settings After Joomla Menu Edit

### ISSUE THREAD

**Issue by N8Solutions** [gantry/gantry5#3208]
Hello @hexplor,

I believe I've found another bug, this time with the Gantry 5 Menu options. I'm encountering a recurring issue with the 'Disable Dropdowns' setting in Gantry 5 (v5.5.17) on a Joomla 4 site. Every time I disable the dropdown for a menu item in Gantry 5 and subsequently edit and save the same menu item in Joomla's menu system, the 'Disable Dropdowns' setting automatically gets unchecked. This issue happens consistently and none of the standard troubleshooting steps, including cache clearing or browser changes, have resolved it.

This problem significantly disrupts the workflow, as it requires me to repetitively reapply the Gantry 5 setting after each Joomla menu edit. Given the nature of the issue, it seems to be a bug where Joomla menu edits are overriding Gantry 5 settings.

Additionally, I noticed a few other menu-related issues on the GitHub repository that might be interconnected:
1. Issue [#3127](https://github.com/gantry/gantry5/issues/3127): Menu items in multiple columns reorder themselves when a new item is added, confirmed as a bug affecting various browsers.
2. Issue [#3154](https://github.com/gantry/gantry5/issues/3154): Changes made to the Gantry Menu layout are not being saved or reflected.
3. Issue [#3207](https://github.com/gantry/gantry5/issues/3207): Custom page classes are not loading on category blog menu items in Joomla 4 with the new template.

Considering these other open issues, there seems to be a pattern of Gantry 5 menu settings not behaving as expected in Joomla 4. This might indicate a broader issue with the integration between Gantry 5 and Joomla 4 menu settings.

Thank you for your attention to this matter. I'm looking forward to any guidance or updates regarding a potential fix.

Warm regards,

Michael

**Comment by hexplor** [comment]
Hello @N8Solutions,
First, apologies for a long delay. I was trying to release last G5 theme with SCSS fixes. 
Anyway, I was trying to recreate your issue on both Joomla 4 (Horizon) and Joomla 5 (Helium) and no go :( Any chance for a test site or step by step-to-reproduce guide? Is your installation fresh one or upgrade? What PHP version are you using?
I have a plan for quick gantry upgrade, so looking forward to fixing this one.
Thanks for your cooperation :)

**Comment by N8Solutions** [comment]
@hexplor no worries, I know you're busy. I will see what I can do and try to get back to you this evening.

**Comment by N8Solutions** [comment]
Hey @hexplor, gather 'round for a classic tale of tech-tragedy, starring yours truly. So, there I was, juggling tabs like a circus clown on espresso, when I stumbled upon a shocking revelation: the problem was a PEBCAK error. Yep, that's right – Problem Exists Between Chair And Keyboard, starring me, the unintentional saboteur.

Here's the play-by-play: I'm there, multitasking like a pro – or so I thought – flipping between the Gantry 5 admin and some other tabs. I'm making changes here, tinkering there, feeling like a tech wizard. But alas, my wizardry was more 'muggle' than magic.

Turns out, while I'm doing my digital dance, changing settings left and right, I missed the plot twist. While I did disable the dropdowns in the Gantry 5 component – plot twist – I undid it later when I hopped back to the other tab, where the menu item is having a party, I save changes there too. But guess what? The dropdowns are still living their best life on the website.

Why? Because in this riveting saga of tab-tastic confusion, I didn't realize that the 'disable dropdowns' option was also under the newly placed "Gantry Menu" tab as a sneaky little checkbox, unchecked, and undoing my so-called 'fixes' in the Gantry 5 backend.

In the end, I facepalmed so hard I could've won an Olympic medal. The lesson? Too many tabs, not enough attention. And maybe, just maybe, I need to slow down and remember that sometimes, the biggest tech issue is the human trying to be a multitasking maestro.

So, as I bring this epic saga of tab-hopping tomfoolery to a close, I realize I might've taken a bit of your time on this wild ride. For that, I offer my sincerest apologies wrapped in a bit of self-mockery. I promise, next time I'll try to keep my inner tech wizard in check (or at least make sure he reads the manual). Thanks for sticking with me through this comedic confessional. Here's to fewer tabs, more focus, and maybe a tech course or two for yours truly. Until next time, keep your dropdowns safe and your keyboards closer!
